test/e2e/specs/multi-block-selection.test.js

```diff
@@ -19,6 +19,7 @@ describe( 'Multi-block selection', () => {
 		// Creating test blocks
 		await page.click( '.editor-default-block-appender' );
 		await page.keyboard.type( 'First Paragraph' );
+		await page.keyboard.press( 'Enter' );
```

Comment:
Why didn't we need this line before?

Reply:
Before, the standard blocks menu would render when the menu was clicked in the next line. However, in this branch, the Inline Blocks menu would render, preventing the standard Image block from being selected on line 24. So we need to press Enter here, to leave the paragraph, so that the standard blocks menu will render and the normal image block can be selected.